tony-core/src/main/java/com/linkedin/tony/tensorflow/TonySession.java

```diff
@@ -204,7 +195,13 @@ public synchronized TonyTask getAndInitMatchingTask(long allocationRequestId) {
       TonyTask[] tasks = entry.getValue();
       for (int i = 0; i < tasks.length; i++) {
         if (tasks[i] == null) {
-          tasks[i] = new TonyTask(jobName, String.valueOf(i), sessionId);
+          String taskCommand = taskCmd;
+          if (tonyConf.get(TonyConfigurationKeys.getExecuteCommandKey(jobName),
+                  tonyConf.get(TonyConfigurationKeys.getContainerExecuteCommandKey())) != null) {
+            taskCommand = tonyConf.get(TonyConfigurationKeys.getExecuteCommandKey(jobName),
```

Comment:
Same comment as above -- please use a local variable for readability and to avoid repeating yourself.